test/integration/base_integration_test.h

```diff
@@ -109,6 +109,26 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
   void registerPort(const std::string& key, uint32_t port);
   uint32_t lookupPort(const std::string& key);
 
+  bool forceCreationOfClusterTrafficStats(absl::string_view cluster_name) {
```

Comment:
doxygen saying it returns false if the cluster is not found.

Reply:
done